src/Packetery/Module/Order/LabelPrint.php

```diff
@@ -312,6 +312,9 @@ private function requestPacketaLabels( int $offset, array $packetIds ): ?Respons
 		if ( ! $response->hasFault() ) {
 			foreach ( array_keys( $packetIds ) as $orderId ) {
 				$order = $this->orderRepository->getById( $orderId );
+				if ( null === $order ) { // Will not happen, collection was already limited.
```

Comment:
Co se nestane? Pokud je podmínka zbytečná tak prosím odstranit.

Reply:
Je potřeba ošetřit, že getById může vrátit null. Komentář jsem upravil.